tests/acceptance/features/webUISharingNotifications/notificationLink.feature

```diff
@@ -1,25 +0,0 @@
-@app-required @notifications-app-required @ocis-reva-issue-14
```

Comment:
According to this comment https://github.com/owncloud/web/issues/5227#issuecomment-1061683197, there won't be a link in the notification so this test doesn't seem relevant anymore